src/mesh/Draco_Mesh.cc

```diff
@@ -605,8 +632,15 @@ void Draco_Mesh::compute_node_to_cell_linkage(
       const std::array<unsigned, 2> node_nbrs = {cellnodes_per_serial_per_rank[rank][3 * i + 1],
                                                  cellnodes_per_serial_per_rank[rank][3 * i + 2]};
 
-      // accumulate local cells (for rank) adjacent to this global node
+      // accumulate local cells and neighbor nodes (for rank) adjacent to this global node
       ghost_dualmap_per_rank[rank][global_node].push_back(std::make_pair(local_cell, node_nbrs));
+
+      // accumulate neighbor node coordinates (in same order as node indices about global_node)
+      const std::array<double, 2> crd_nbr1 = {coord_nbrs_per_serial_per_rank[rank][4 * i],
+                                              coord_nbrs_per_serial_per_rank[rank][4 * i + 1]};
+      const std::array<double, 2> crd_nbr2 = {coord_nbrs_per_serial_per_rank[rank][4 * i + 2],
+                                              coord_nbrs_per_serial_per_rank[rank][4 * i + 3]};
+      ghost_coord_nbrs_per_rank[rank][global_node].push_back(std::make_pair(crd_nbr1, crd_nbr2));
```

Comment:
Does `emplace_back` work here?

Reply:
Yes - it should work here - thank you catching this!

Reply:
These `push_back` calls have been replaced with `emplace_back`.